frontend/src/lib/lemon-ui/Popover/Popover.scss

```diff
@@ -94,72 +88,45 @@
     .Popover--max-content-width & {
         width: max-content;
     }
-
-    .posthog-3000 &,
-    .posthog-3000 .Popover--actionable & {
-        padding: 0.25rem;
-        background: var(--bg-light);
-        border-color: var(--secondary-3000-button-border);
-    }
 }
 
 .Popover__arrow {
     position: absolute;
     width: 0.5rem;
     height: 0.5rem;
-    background: var(--bg-3000);
+    background: var(--bg-light);
     transform: rotate(45deg);
 
-    .posthog-3000 & {
-        background: var(--bg-light);
-    }
-
     [data-placement^='bottom'] & {
-        top: -0.25rem;
+        top: -0.3125rem;
         border-top: 1px solid var(--border);
         border-left: 1px solid var(--border);
-
-        .posthog-3000 & {
-            top: -0.3rem;
-        }
```

Comment:
Been wondering why the increase in offset in 3000? @daibhin

Reply:
It had to do with this PR: https://github.com/PostHog/posthog/pull/18867. Afaik it was because the bottom border on popovers was thicker so the caret needed to be shifted down slightly further to line up with the edge of the component. The timezone component is a good one to see the caret on a popover